venv_update.py

```diff
@@ -463,7 +463,11 @@ def do_install(reqs):
         pip(('uninstall', '--yes') + tuple(sorted(extraneous)))
 
     # Cleanup: remove stale values from the cache and wheelhouse that have not been accessed in a week.
-    run(['find', pip_download_cache, pip_wheels, '-not', '-atime', '-7', '-exec', 'rm', '-rf', '{}', '+'])
```

Comment:
rm -rf should already be reentrant.
It's just find that needs to be fixed.
There's actually a find option exactly for this:

```
-ignore_readdir_race
              Normally, find will emit an error message when it fails to stat a file.  If you give this option and a file is deleted
              between  the  time find reads the name of the file from the directory and the time it tries to stat the file, no error
              message will be issued.
```

That said, the except/pass is probably still a good idea for those with old/non-existent `find`s (osx/windows).